x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/parent_process_draggable.tsx

```diff
@@ -15,13 +15,26 @@ interface Props {
   contextId: string;
   endgameParentProcessName: string | null | undefined;
   eventId: string;
+  processParentPid: number | null | undefined;
+  processParentName: string | null | undefined;
   processPpid: number | undefined | null;
   text: string | null | undefined;
 }
 
 export const ParentProcessDraggable = React.memo<Props>(
-  ({ contextId, endgameParentProcessName, eventId, processPpid, text }) => {
-    if (isNillEmptyOrNotFinite(endgameParentProcessName) && isNillEmptyOrNotFinite(processPpid)) {
+  ({
+    contextId,
+    endgameParentProcessName,
+    eventId,
+    processParentName,
+    processParentPid,
+    processPpid,
+    text,
+  }) => {
+    if (
```

Comment:
I am wondering if you really want that because I can see in this component we have other condition in the render for the props `text`, `processParentPid` and `processPpid`

Reply:
Talk to @andrew-goldstein, that's all good and valid